samples/sampler/stories/qa/Marquee.js

```diff
@@ -168,6 +168,39 @@ class MarqueeWithContentChanged extends Component {
 	}
 }
 
+class MarqueeSynchronizedWithContentChanged extends Component {
+	constructor (props) {
+		super(props);
+		this.state = {
+			count: 0
+		};
+	}
+
+	handleClick = () => {
+		this.setState(({count}) => ({count: ++count % 3}));
+	};
+
+	render () {
+		return (
+			<div>
+				<ol>
+					<li>Click once to show the ellipsis just before the text marquees the first time.</li>
+					<li>
+						Click a second time to show the ellipsis just before the text marquees the first time
+					</li>
+					<li>Click again to return to a short string without marquee.</li>
+				</ol>
+				<Button onClick={this.handleClick}>{'Click Me'}</Button>
+				<Controller>
+					<Marquee style={{width: '800px'}} marqueeOn={'render'}>
```

Comment:
Let's use `ri.ScaleToRem` for the number.

Reply:
It's fixed.